astroquery/vo_conesearch/validator/tstquery.py

```diff
@@ -65,10 +78,10 @@ def parse_cs(ivoid, cap_index=1):
 
             d = OrderedDict({'RA': ra, 'DEC': dec, 'SR': sr})
 
-        except Exception:  # pragma: no cover
+        except Exception as e:  # pragma: no cover
```

Comment:
I slightly prefer calling it `exc` or `err` rather than the one letter name, but it's not worth holding up merging with this.

Reply:
Ops... Should I open a follow-up PR? `e` is from an old habit before PEP 8 wanted 3 characters.